### PR TITLE
Add support for École supérieure privée d'ingénierie et de technologie

### LINKED ISSUE #422: [New University]: ESPRIT engineering college
### University name

École Supérieure Privée d'Ingénierie et de Technologies

### University abbreviation

ESPRIT

### University country

تونس

### URL of university Blackboard instance

https://esprit.blackboard.com/

### URL after logging in

https://esprit.blackboard.com/ultra

### University network information

Name: ORG-ATIA2-AFRINIC
Description: missing
Country: TN

### Valid Blackboard data sources (optional)

_No response_

### Anything else to add?

Full report:
```
{"rdapConformance":["rdap_level_0","nro_rdap_profile_0","cidr0"],"notices":[{"title":"ABOUT","description":["This is the AfriNIC RDAP server."],"links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"describedby","href":"https://www.afrinic.net/support/whois-db/reference-manual","hreflang":["en"],"type":"text/html","title":"AFRINIC Database Reference Manual","media":"screen"}]},{"title":"Terms and Conditions","description":["This is the AFRINIC Database query service. The objects are in RDAP format."],"links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"terms-of-service","href":"https://afrinic.net/whois/terms","type":"text/html"}]},{"title":"Whois Inaccuracy Reporting","description":["If you see inaccuracies in the results, please visit: "],"links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"inaccuracy-report","href":"https://www.afrinic.net/support/whois-db/reference-manual","type":"text/html","title":"AFRINIC WHOIS Inaccuracy Report"}]},{"title":"jCard sunset end","description":["2023-12-31T23:59:59Z"],"links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"related","href":"https://www.ietf.org/id/draft-ietf-regext-rdap-jscontact-12.html","type":"text/html","title":"jCard deprecation and replacement with JSContact Card"},{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"alternate","href":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255?jscard=1","type":"application/rdap+json","title":"JSContact Card version"}]},{"title":"jCard deprecation end","description":["2022-12-31T23:59:59Z"],"links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"related","href":"https://www.ietf.org/id/draft-ietf-regext-rdap-jscontact-12.html","type":"text/html","title":"jCard deprecation and replacement with JSContact Card"},{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"alternate","href":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255?jscard=1","type":"application/rdap+json","title":"JSContact Card version"}]}],"handle":"196.203.180.0 - 196.203.183.255","lang":"en","links":[{"value":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255","rel":"self","href":"https://rdap.afrinic.net/rdap/ip/196.203.180.0 - 196.203.183.255"}],"events":[{"eventAction":"registration","eventDate":"2013-10-16T11:10:00Z"},{"eventAction":"last changed","eventDate":"2013-10-16T12:05:00Z"}],"status":["active"],"port43":"whois.afrinic.net","name":"ORG-ATIA2-AFRINIC","type":"SUB-ALLOCATED PA","country":"TN","entities":[{"handle":"ORG-ATIA2-AFRINIC","lang":"en","remarks":[{"title":"Remark","description":["data has been transferred from RIPE Whois Database 20050221"]}],"links":[{"value":"https://rdap.afrinic.net/rdap/entity/ORG-ATIA2-AFRINIC","rel":"self","href":"https://rdap.afrinic.net/rdap/entity/ORG-ATIA2-AFRINIC"}],"status":["active"],"port43":"whois.afrinic.net","vcardArray":["vcard",[["version",{},"text","4.0"],["kind",{},"text","org"],["fn",{},"text","ATI - Agence Tunisienne Internet"],["tel",{"type":"work"},"text","tel:+216-71-846-100"],["tel",{"type":"work"},"text","tel:+216-70-147-700"],["tel",{"type":"work"},"text","tel:+216-71-843-843"],["email",{},"text","lir@ati.tn"],["email",{},"text","adel@ati.tn"],["email",{},"text","mohamed.mnif@ati.tn"],["adr",{"label":"13, rue Jughurta, Belvedere\nTunis 1002"},"text",["13, rue Jughurta, Belvedere","Tunis 1002","","","","",""]]]],"roles":["registrant"],"objectClassName":"entity"},{"handle":"AK34-AFRINIC","lang":"en","links":[{"value":"https://rdap.afrinic.net/rdap/entity/AK34-AFRINIC","rel":"self","href":"https://rdap.afrinic.net/rdap/entity/AK34-AFRINIC"}],"status":["active"],"port43":"whois.afrinic.net","vcardArray":["vcard",[["version",{},"text","4.0"],["kind",{},"text","individual"],["fn",{},"text","Ahmed Kooli"],["tel",{"type":"work"},"text","tel:+216-71-780-900"],["email",{},"text","Ahmed@topnet.tn"],["adr",{"label":"Centre Urbain Nord\n1073 Tunis\nTN"},"text",["Centre Urbain Nord","1073 Tunis","TN","","","",""]]]],"roles":["administrative","technical"],"objectClassName":"entity"}],"startAddress":"196.203.180.0","endAddress":"196.203.183.255","ipVersion":"v4","parentHandle":"196.203.0.0 - 196.203.255.255","cidr0_cidrs":[{"v4prefix":"196.203.180.0","length":22}],"objectClassName":"ip network"}
```

### License

- [X] I agree to follow this project's License

### PR DESCRIPTION
**Pull request by sanjacob**
Created on behalf of @TheLime1
Closes #422

### CODE REVIEW

**Comment by TheLime1** [comment]
i test it , the download process works fine

File: blackboard_sync/universities.json [hunk 1]
```diff
@@ -670,6 +670,16 @@
     },
     "api_url": "https://mstc.blackboard.com"
   },
+  {
+    "name": "École supérieure privée d'ingénierie et de technologie",
```
Comment (TheLime1) [inline blackboard_sync/universities.json:674]: There is only a small problem: your app is not compatible with special French characters.
So, I suggest renaming it to: `Ecole superieure privee d'ingenierie et de technologie`

**Review by TheLime1 — CHANGES_REQUESTED** [review]

File: blackboard_sync/universities.json (hunk 1 above)
Comment (sanjacob) [reply blackboard_sync/universities.json:674]: What do you mean it is not compatible? Is there an error with the UI?

**Comment by sanjacob** [comment]
@TheLime1 Could you let me know the error you are seeing regarding the french characters?

**Comment by TheLime1** [comment]
image

**Comment by TheLime1** [comment]
I think it's an encoding issue, but the download process works

**Comment by sanjacob** [comment]
I have made #427 which should solve that, after I merge you can edit your issue so that this PR is re-created. You could make a simple edit such as adding one space to the "additional information" field. I will let you know when I have merged #427.

**Comment by TheLime1** [comment]
i was gonna test it right now hahah , thank you :)

**Comment by TheLime1** [comment]
image


Fixed

**Review by TheLime1 — APPROVED** [review]

**Comment by sanjacob** [comment]
@TheLime1 Just one more thing before I merge this. All other universities have their name in Title Case, so it would be a good idea to follow this format. However, I'm not sure how this would look given that it's in French. From their website and social media it is similar to what you provided but some of them have "Technologies" in plural rather than singular. Moreover they have written "Ecole" without the accent ([example](https://www.instagram.com/esprit_ingenieur/)). Just edit the university name to be in the correct format in Title Case and I'll merge it.

**Comment by TheLime1** [comment]
i dont have direct editing rights here , so i made #428

**Comment by sanjacob** [comment]
You can edit the issue directly and the PR will also update.

**Comment by TheLime1** [comment]
@sanjacob you can take a look

**Comment by sanjacob** [comment]
@TheLime1 So is the accent on the first letter optional? Just curious

**Comment by TheLime1** [comment]
In French, it's rare to use accents on uppercase letters, but both are correct

**Pull request merged**